jwst/ramp_fitting/tests/test_ramp_fit.py

```diff
@@ -331,8 +331,8 @@ def test_two_groups_fit(self, method):
         np.testing.assert_allclose(slopes[0].data[50, 50], cds_slope, 1e-6)
         #expect SATURATED
         assert slopes[0].dq[50, 51] == 2 # is there a better way to do this test?
-        #expect SATURATED since 1st group is Saturated
-        assert slopes[0].dq[50, 52] == 2 # is there a better way to do this test?
+        #expect SATURATED and DO_NOT_USE, because 1st group is Saturated
+        assert slopes[0].dq[50, 52] == 3 # is there a better way to do this test?
```

Comment:
For readability, one can use the `datamodels.dqflags.pixel` or `group` to give meaning to the value `3`.

Reply:
Updated in latest commit.